yt/visualization/volume_rendering/lens.py

```diff
@@ -180,8 +180,8 @@ def _get_sampler_params(self, camera, render_source):
         north_vec = camera.unit_vectors[1]
         normal_vec = camera.unit_vectors[2]
 
-        px = np.mat(np.linspace(-0.5, 0.5, camera.resolution[0]))
-        py = np.mat(np.linspace(-0.5, 0.5, camera.resolution[1]))
+        px = np.linspace(-0.5, 0.5, camera.resolution[0])
+        py = np.linspace(-0.5, 0.5, camera.resolution[1])
```

Comment:
This is one of two code changes. It prevents a warning about the fact `mat` shouldn't be used.